proposal/random-with-units.md

```diff
@@ -0,0 +1,101 @@
+# Random: Draft 1
+
+*([Issue](https://github.com/sass/sass/issues/1890))*
+
+This proposal defines the behavior of the built-in [`random()`][random]
```

Comment:
Nit: "Defines" suggests that this defines an entirely new function. It would be good to be a bit more specific about what behavior it's changing.

Reply:
Right, so I had wording like that, but then realized that in a way it was never "defined" here and only in the Sass website, but I can update it to clarify it's modifying the current behavior